/mission_metrics:
  ros__parameters:
    result_path: "~/suave/results" #Path to save results

/suave_rosa_bt:
  ros__parameters:
    time_limit: 300 #Time limit for a time_constrained mission

/water_visibility_observer_node:
  ros__parameters:
    water_visibility_period: 120 #Water visibility period in seconds
    water_visibility_min: 1.25 #Minimum value for water visibility
    water_visibility_max: 3.75 #Maximum value for water visibility
    water_visibility_sec_shift: 0.0 #Water visibility seconds shift to left

/thruster_monitor:
  ros__parameters:
    thruster_events: # Thruster events, format: (thruster number (1 to 6), failure or recovery, delta time in seconds(from the last event))
    - (1,failure,60)
